test/parallel/test-debug-usage.js

```diff
@@ -24,5 +24,8 @@ child.on('exit', common.mustCall(function(code) {
   const outputLines = actualUsageMessage.split('\n');
   assert.strictEqual(code, 1);
   for (let i = 0; i < expectedLines.length; i++)
-    assert(expectedLines[i].test(outputLines[i]));
+    assert.ok(
+      expectedLines[i].test(outputLines[i]),
+      `${outputLines[i]} did not match ${expectedLines[i]}`
+    );
```

Comment:
Unrelated change/commit?

Reply:
Looks like it's fromhttps://github.com/nodejs/node/pull/30913.

Reply:
Whoops, though I pulled that out....

Reply:
Rebased out.